wordpress-mobile/WordPress-Aztec-iOS
# Move selection after pasting text.

Fixes #301 

**To test:**

Just paste something and make sure the caret position is moved to the end of the pasted content.
